hooks/useMapLayer.ts

```diff
@@ -11,22 +11,28 @@ import {ZodError} from 'zod';
 
 export const useMapLayer = (center_id: string) => {
   const clientProps = React.useContext<ClientProps>(ClientContext);
-  return useQuery<MapLayer, AxiosError | ZodError>(['map-layer', center_id], async () => {
```

Comment:
I have no idea why `prettier` decided to entirely reformat this file - @floatplane any thoughts?

Reply:
I'm guessing that once you added a 3rd argument to `useQuery`, prettier decided it had to break the arguments and align them?